src/fields/NullableField.ts

```diff
@@ -22,13 +22,9 @@ export class NullableField<T extends FieldModel<unknown>> extends FieldModel<Nul
      * @returns {boolean}
      */
     override validate(value: unknown): true | never {
-        if (value) {
-            return this.model.validate(value);
-        }
-
-        if (![null, undefined].includes(value)) {
-            throw new TypeError(`value '${value}' is not 'null' or 'undefined'`);
-        }
+        if (value) return this.model.validate(value);
+        if (value !== null) throw new TypeError(`value '${value}' is not 'null' or 'undefined'`);
```

Comment:
Allow `undefined`

Reply:
Yeah but the eslint check wasn't allowing but lemme change it now.